.github/workflows/dotnet.yaml

```diff
@@ -59,6 +59,30 @@ jobs:
       - name: Run Fantomas
         run: ./hooks/pre-push
 
+  analyze-code:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup .NET SDK v6.0.x
```

Comment:
Out of interest, is the net6 SDK required here? Could it be net7 or net8 without problems? (I'm fine with net6, I just am interested in the answer.)

Reply:
No, that's not a strict requirement. net7 and net8 should also work. Locally I tested with net8.